### PR TITLE
Added Visualiser and Song URL paramter.

### PR DESCRIPTION
**Pull request by Zeludon**
[Try it!](http://zeludon.me/takeb1nzyto.space/?song=9)

### CODE REVIEW

File: assets/music.js [hunk 1]
```diff
@@ -15,12 +15,18 @@ window.onload = function() {
       ['Iseeicy - TIME', 'https://www.youtube.com/watch?v=nw2qiR6f05Q'],
       ['Kai Engel - The Flames of Rome', 'https://www.youtube.com/watch?v=Ck-ZYR-etgI'],
       ['NOMA - Brain Power', 'https://www.youtube.com/watch?v=mj-v6zCnEaw'],
-      ['Renard - Send It To The Moon', 'https://www.youtube.com/watch?v=J0vUBlcGelI'],
-      ['Teo Wei Yong - Divided Singularity', 'https://www.youtube.com/watch?v=kvJYs14j0bg']
+      ['Renard - Send It To The Moon', 'https://www.youtube.com/watch?v=J0vUBlcGelI'],
```
Comment (deansheather) [inline assets/music.js]: _But why?_

**Comment by deansheather** [comment]
I noticed that the `Visualize` button doesn't toggle the visualizer, can you quickly chuck that in? Then I'll be happy to squash and merge.

**Comment by Zeludon** [comment]
All fixed.

**Comment by deansheather** [comment]
:+1:

**Pull request merged**